app/services/delete_application.rb

```diff
@@ -0,0 +1,90 @@
+class DeleteApplication
+  include ImpersonationAuditHelper
+
+  attr_reader :actor, :application_form, :ticket
+
+  APPLICATION_FORM_FIELDS_TO_REDACT = %i[
+    first_name
+    last_name
+    first_nationality
+    second_nationality
+    english_main_language
+    english_language_details
+    other_language_details
+    date_of_birth
+    further_information
+    phone_number
+    address_line1
+    address_line2
+    address_line3
+    address_line4
+    country
+    postcode
+    disability_disclosure
+    uk_residency_status
+    work_history_explanation
+    becoming_a_teacher
+    subject_knowledge
+    interview_preferences
+    work_history_breaks
+    volunteering_experience
+    equality_and_diversity
+    safeguarding_issues
+    international_address
+    right_to_work_or_study
+    right_to_work_or_study_details
+    third_nationality
+    fourth_nationality
+    fifth_nationality
+    feedback_satisfaction_level
+    feedback_suggestions
+    work_history_status
+  ].freeze
```

Comment:
Considering that the application is anonymised, have we considered keeping non sensitive data for statistical reasons? E.g. safeguarding issues, right to work or study, satisfaction level, disability etc?

Reply:
The problem with free-text fields is you can't really know what's in them and whether they include information that could be used to identify the candidate. Probably best to err on the side of caution here. Impact on stats should be minimal, as we've only ever received a few deletion requests. We retain their application choices (the courses they've tried to apply to), their support reference and timestamps so should still be able to produce some basic metrics. Things like safeguarding information, disability etc. sound problematic to retain, especially as we store the Zendesk link to the deletion request.